flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/metadata/FlinkRelMdWindowProperties.scala

```diff
@@ -252,13 +254,40 @@ class FlinkRelMdWindowProperties private extends MetadataHandler[FlinkMetadata.W
   }
 
   def getWindowProperties(
-      rel: StreamPhysicalWindowAggregate,
+      rel: StreamPhysicalWindowAggregateBase,
       mq: RelMetadataQuery): RelWindowProperties = {
+    rel match {
+      case _: StreamPhysicalWindowAggregate =>
+        val aggregate = rel.asInstanceOf[StreamPhysicalWindowAggregate]
+        getWindowAggregateWindowProperties(
+          aggregate.grouping.length + aggregate.aggCalls.size(),
+          aggregate.namedWindowProperties,
+          aggregate.windowing.getWindow,
+          aggregate.windowing.getTimeAttributeType
+        )
+      case _: StreamPhysicalGlobalWindowAggregate =>
+        val aggregate = rel.asInstanceOf[StreamPhysicalGlobalWindowAggregate]
+        getWindowAggregateWindowProperties(
+          aggregate.grouping.length + aggregate.aggCalls.size(),
+          aggregate.namedWindowProperties,
+          aggregate.windowing.getWindow,
+          aggregate.windowing.getTimeAttributeType
+        )
+      case _ =>
```

Comment:
Can the input of window join be `StreamPhysicalGroupWindowAggregateBase`? If the answer is yes, it may need to add the case of `StreamPhysicalGroupWindowAggregateBase`.